prefect_aws/ecs.py

```diff
@@ -427,28 +427,32 @@ def cloudwatch_logs_options_requires_configure_cloudwatch_logs(
             )
         return values
 
-    @root_validator
+    @root_validator(pre=True)
     def image_is_required(cls, values: dict) -> dict:
         """
-        Enforces that an image is available if the user sets it to `None`.
+        Enforces that an image is available if image is `None`.
         """
-        if (
-            not values.get("image")
-            and not values.get("task_definition_arn")
-            # Check for an image in the task definition; whew!
-            and not (
-                get_prefect_container(
-                    (values.get("task_definition") or {}).get(
-                        "containerDefinitions", []
-                    )
-                )
-                or {}
-            ).get("image")
-        ):
-            raise ValueError(
-                "A value for the `image` field must be provided unless already "
-                "present for the Prefect container definition a given task definition."
+        has_image = values.get("image")
+        has_task_definition_arn = values.get("task_definition_arn")
```

Comment:
Based on these variable names, I'd expect these to be boolean values.
```suggestion
        has_image = bool(values.get("image"))
        has_task_definition_arn = bool(values.get("task_definition_arn"))
```